charts/gardener-extension-shoot-oidc-service/templates/deployment.yaml

```diff
@@ -82,6 +82,9 @@ spec:
         - --gardener-version={{ .Values.gardener.version }}
         - --metrics-bind-address=:{{ .Values.metrics.port }}
         - --health-bind-address=:{{ .Values.healthPort }}
+        {{- if .Values.gardener.autonomousShootCluster }}
+        - --autonomous-shoot-cluster={{ .Values.gardener.autonomousShootCluster }}
```

Comment:
As an extension maintainer, I am not sure what is the actual implication of this flag. What are the requirements for an extension to work with autonomous shoot clusters?
And when I write "work with autonomous shoot clusters", I am asking myself is it:
- working inside autonomous shoot cluster as a workload, or
- extending the autonomous shoot capabilities with the oidc webhook authenticator, or
- something else?

Probably the flag name can be improved a little to be more clear what is its intention.